test/Mvc6Framework45.FunctionalTests/project.json

```diff
@@ -56,12 +56,8 @@
   ],
   "scripts": {
     "postrestore": [
-      "npm install",
-      "node_modules/.bin/flatten-packages", 
-      "node_modules/.bin/bower install"
```

Comment:
Why have we removed this? We specifically added this earlier because the CI builds fail due to some NPM packages having dependency folder trees that are too deep and break the 256 character limit enforced by the .NET APIs the build agent uses. Unless something changed (such as DNX flattening Node packages automatically or Bower dependencies becoming more shallow) we should keep this logic.